docs/csharp/whats-new/tutorials/static-abstract-interface-methods.md

```diff
@@ -0,0 +1,147 @@
+
+You can create a structure that creates a string of 'A' characters where each increment adds another character to the string using the following code:
+
+:::code language="csharp" source="./snippets/staticinterfaces/RepeatSequence.cs":::
+
+More generally, you can build any algorithm where you might want to define `++` to mean "produce the next value of this type". Using this interface produces clear code and results:
+
+:::code language="csharp" source="./snippets/staticinterfaces/Program.cs" id="TestRepeat":::
+
+The preceding example produces the following output:
+
+```powershell
+A
+AA
+AAA
+AAAA
+AAAAA
+AAAAAA
+AAAAAAA
+AAAAAAAA
+AAAAAAAAA
+AAAAAAAAAA
+```
+
+This small example demonstrates the motivation for this feature. You can use natural syntax for operators, constant values, and other static operations. You can explore these techniques when you create multiple types that rely on static members, including overloaded operators. Define the interfaces that match your types' capabilities and then declare those types' support for the new interface.
+
+## Generic math and experimental features
+
+The motivating scenario for allowing static methods, including operators, in interfaces is to support *generic math* algorithms. The [System.Runtime.Experimental](https://www.nuget.org/packages/System.Runtime.Experimental/) NuGet package contains interface definitions for many arithmetic operators, and derived interfaces that combine many arithmetic operators in an `INumber<T>` interface. Let's apply those types to build a `Point<T>` class that can use any numeric type for `T`. The point can be moved by some `XOffset` and `YOffset` using the `+` operator.
+
+Start by creating a new Console application, either by using `dotnet new` or Visual Studio. Next, add the `System.Runtime.Experimental` NuGet package to your project. You'll need to enable preview features to use the types in this package. Enabling preview features also sets the C# language version to "preview", which enables C# 10 preview features. Add the following element to your *csproj* file:
+
+```xml
+<EnablePreviewFeatures>true</EnablePreviewFeatures>
+```
+
+> [!NOTE]
+> This element cannot be set using the Visual Studio UI. You need to edit the project file directly.
+
+The public interface for the `Translation<T>` and `Point<T>` should look like the following code:
+
+```csharp
+// Note: Not complete. This won't compile yet.
+public record Translation<T>(T XOffset, T YOffset);
+
+public record Point<T>(T X, T Y)
+{
+    public static Point<T> operator +(Point<T> left, Translation<T> right);
+}
+```
+
+You use the `record` type for both the `Move<T>` and `Point<T>` types: Both store two values, and they represent data storage rather than sophisticated behavior. The implementation of `operator +` would look like the following code:
```

Comment:
```suggestion
You use the `record` type for both the `Translation<T>` and `Point<T>` types: Both store two values, and they represent data storage rather than sophisticated behavior. The implementation of `operator +` would look like the following code:
```